index.html

```diff
@@ -39,7 +49,7 @@
           <h2></h2>
           <time pubdate datetime="2000-01-01"></time>
         <section class="article-body"></section>
-        <p class="read-on">Read More...</p>
+        <a class="read-on">Read On &rarr;</a>
```

Comment:
Why do you have this a tag? doesn't show up on the page.  Should try and use a p tag.